### ISSUE TITLE
Create a list of ops files that can be moved to cf-deployment

### ISSUE THREAD

**Issue by viovanov** [SUSE/kubecf#11]
We should compile a list of ops files that are generic and can be moved to cf-deployment, then make PRs upstream.

**Comment by fargozhu** [comment]
@viovanov issue superseded by #12?

**Comment by viovanov** [comment]
@fargozhu no, that one is about patches. Ops files mutate the deployment manifest, so it does things differently. Some of these ops files could be part of `cf-deployment` - we just need to figure out which.
